git/menu-windows.json

```diff
@@ -0,0 +1,13 @@
+{
+    "menu_name": "Anaconda${PY_VER} ${PLATFORM}",
+    "menu_items":
+    [
+    {
+        "name": "Git Bash",
+        "script": "${PREFIX}/Library/bin/bash.exe",
+        "scriptarguments": ["--login", "-i", "--"],
+        "workdir": "${PERSONALDIR}",
+        "icon": "${MENU_DIR}/git-for-windows.ico"
```

Comment:
This is so nice!! I think we are so close to have on Windows the same experience developers have on Unix, thanks to conda! :-)